.github/workflows/comment-deploy-url-to-pr.yaml

```diff
@@ -0,0 +1,19 @@
+name: Comment deploy URL to PR
+
+on:
+    pull_request:
+        types: [opened]
+
+jobs:
+    comment-on-pr:
+        runs-on: ubuntu-latest
+        name: Comment deploy URL to PR
+        steps:
+            - name: Get branch hash
+              id: get_hash
+              run: echo "::set-output name=hash::$(echo '${{ github.ref }}' | shasum | awk '{print substr($1,0,36)}')"
```

Comment:
I think some comment here, pointing to the other workflow where the same hash is generated, would be useful. (And maybe in the other direction as well, instead of “a staging area”.)

Reply:
Done.